adaptor/elasticsearch/clients/v5/writer.go

```diff
@@ -90,10 +91,47 @@ func (w *Writer) Write(msg message.Msg) func(client.Session) (message.Msg, error
 			w.bp.Flush()
 			br = elastic.NewBulkDeleteRequest().Index(w.index).Type(indexType).Id(id)
 		case ops.Insert:
-			br = elastic.NewBulkIndexRequest().Index(w.index).Type(indexType).Id(id).Doc(msg.Data())
+			if _, ok := msg.Data()["elastic_parent"]; ok {
```

Comment:
can we make this field configurable via [ClientOptions](https://github.com/compose/transporter/blob/a25a8e1daa2bd7889c145b6a1893f89df8fa4dde/adaptor/elasticsearch/clients/registry.go#L29-L35) with a default of `elastic_parent`?